### ISSUE TITLE
"Origin of Electricity in the last 24 hours" curve not showing for Aruba

### ISSUE THREAD

**Issue by Hyric89** [tmrowco/electricitymap-contrib#2282]
Hello

Something peculiar is going on with Aruba. Normally, when data is missing, both the "Carbon intensity" and "Origin of electricity" curves go missing. Here, I can see the 24-hour Carbon intensity curve just fine but the 24-hour Origin of electricity curve is blank. This is despite the Electricity consumption by source bar chart showing data.

Aruba's data source seems to be working just fine and ElectricityMap's figures according to the electricity consumption by source. https://webaruba.com/renewable-energy-dashboard/aruba

Screenshots are included below.
![Screenshot from 2020-03-14 14-49-53](https://user-images.githubusercontent.com/35940453/76688580-7eb64c00-6604-11ea-934a-1740e5ba4ebd.png)
![Screenshot from 2020-03-14 14-57-51](https://user-images.githubusercontent.com/35940453/76688581-84139680-6604-11ea-939e-71cea6b7d744.png)

**Comment by jarek** [comment]
I can replicate, chart is not being shown despite data coming in from server, Firefox 73 / Ubuntu:

![Screenshot_2020-03-14_17-49-19](https://user-images.githubusercontent.com/47415/76690998-2f7c1580-661c-11ea-90dc-53f17d05c0aa.png)

To be honest I think I've had this happen before on other regions, and I've always taken it to be a rendering bug that manifested in Firefox. But I don't have many details.

Debugger console shows the following lines that may or may not be relevant:

```
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://browser.sentry-cdn.com/5.9.1/bundle.min.js. (Reason: CORS request did not succeed).
None of the “sha384” hashes in the integrity attribute match the content of the subresource. www.electricitymap.org
...
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://browser.sentry-cdn.com/5.9.1/bundle.min.js. (Reason: CORS request did not succeed).
None of the “sha384” hashes in the integrity attribute match the content of the subresource. www.electricitymap.org
...
ReferenceError: Sentry is not defined
...
Content Security Policy: The report URI (about:blank) should be an HTTP or HTTPS URI.
Content Security Policy: The page’s settings observed the loading of a resource at blob:https://www.electricitymap.org/cd029639-7b55-4c60-8e4b-eeb94251269f (“worker-src”). A CSP report is being sent.
...
Error: WebGL warning: texImage2D: Alpha-premult and y-flip are deprecated for non-DOM-Element uploads. <32 times> mapbox-gl.js:29:240685
Error: WebGL: No further warnings will be reported for this WebGL context. (already reported 32 warnings)
```

(Sentry scripts are being blocked for me by uMatrix as coming from third-party domain, I don't know if the CSP errors are related)

**Comment by corradio** [comment]
@fbarl could be this due to one of the frontend changes?

**Comment by fbarl** [comment]
> @fbarl could be this due to one of the frontend changes?

Indeed, I managed to reproduce and trace the bug to `getMaxTotalValue` not accounting for `NaN` values (see https://github.com/tmrowco/electricitymap-contrib/pull/2171/files#diff-e666df2f408c2db2102da3e7af69650aR37).

I fixed it in #2285.
